docs/cookbook/installing-on-mac-os-and-windows.md

```diff
@@ -104,6 +104,8 @@ Alternatively, you can also change individual installation parameters in `.env`,
 - `DATABASE_NAME`
 - `DATABASE_HOST`
 - `DATABASE_PORT`
+- `DATABASE_PLATFORM` — defines the driver used by the doctrine to connect to the database (e.g. `mysql` or `pgsql`)
+- `DATABASE_DRIVER` — handles the prefix for distinguishing the database you are connecting to (e.g. `pdo_mysql` or `pdo_pgsql`)
```

Comment:
feels to me it's the other way around, cc @webhdx

Reply:
I applied the change suggested by @webhdx 
So what's the final and correct version? 😄

Reply:
> Well, now it's according to @webhdx 's description in [ezsystems/ezplatform#434](https://github.com/ezsystems/ezplatform/pull/434)

`DATABASE_DRIVER` is for defining driver (as name suggest), `DATABASE_PLATFORM` is for defining prefix. So the descriptions are ok, but just need to be swapped (see the ezsystems/ezplatform#434 description).